docs/rfcs/0003_templates.md

```diff
@@ -0,0 +1,347 @@
+# 🗃 Data Submission Templates
+
+**Status**: Approved, waiting for implementation
+
+This document expands on the [MVP solution](0002_templates_mvp.md#motivation)
+by adding new concepts such as template versioning.
+
+## 📓 Model
+
+![Relational Diagram](../_static/images/template-models.png)
```

Comment:
I would suggest implementing the whole model at once as adding the versions later may make migrations tricky.

Reply:
I think you may want a one-to-one from study to PackageVersion rather than a TemplatePackage. Otherwise, how will you know which version the study is using (unless they always use the latest)?

Reply:
Let's talk about whether we want to support template versioning